Detections/Tenable/Tenable - Low Vulnerability found.yaml

```diff
@@ -0,0 +1,20 @@
+id: 616aecbc-318f-4936-8267-bec55356e0bd
+name: Tenable - Low Vulnerability found
+description: |
+  'This query identifies a Tenable Vulnerability of Low Vulnerability.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Tenable
+    dataTypes: 
+      - Tenable_Assets_List
+queryFrequency: 12h
+queryPeriod: 12h
+triggerOperator: gt
+triggerThreshold: 0
+query: |
+
+  Tenable_Assets_List
+  | join (Tenable_Vulns_List) 
+      on AssetID
+  | project Timestamp=TimeGenerated1, AssetID, FQDN, IP, Operating_system, VulnID, VulnName, plugin_family, SeverityType, Detection_source
+  | where SeverityType in ("Low")
```

Comment:
Generally you will want the where clause above the project for performance reasons.  I don't see any reason to NOT move it above the project here.